### PR TITLE
Retired Message Simulator Java

### LINKED ISSUE #56: Retire Java Message Simulator
Use the Message Simulator in https://github.com/Esri/vehicle-commander-qt instead

### PR DESCRIPTION
**Pull request by garys-esri**
Fixes #56

- Removed Message Simulator Java code
- Linked to geomessage-simulator-qt
- Added documentation

### CODE REVIEW

**Comment by garys-esri** [comment]
@csmoore, will you please review and merge when you have a moment?

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ This section is for developers who just need to quickly build and run the applic
 * Uses ArcGIS Runtime 10.2.4.
 * Java 8 or higher is required. ArcGIS Runtime 10.2.4 requires Java 7 or higher, but Oracle included [a bug fix in Java 8](http://bugs.java.com/bugdatabase/view_bug.do?bug_id=7107883) that allows UDP networking to behave properly for wired and Wi-Fi networks. Vehicle Commander's messaging depends on UDP networking. Therefore, you should use Java 8 or higher to ensure that messaging works properly.
 * Ant 1.9+ is required for building Vehicle Commander using the build script. Java 8 depends on Ant 1.9 or higher.
+* The Java version of Vehicle Commander was retired. Use the [GeoMessage Simulator](https://github.com/Esri/geomessage-simulator-qt) instead.
```
Comment (csmoore) [inline README.md]: ...of Vehicle Commander *geomessage simulator*...

**Comment by csmoore** [comment]
Looks good - just one minor thing noted.

And when you eventually have a standalone deployment of the geomessage-simulator-qt you might want to add that link in.

**Comment by garys-esri** [comment]
@csmoore, thanks for catching my mistake. I corrected it.

I will link to the standalone deployment as part of https://github.com/Esri/geomessage-simulator-qt/issues/19 .

**Pull request merged**